### PR TITLE
Add hasta config for taxprofiler

### PR DESCRIPTION
**Pull request by sofstam**
This PR defines specific parameters for BBMAP_BBDUK and MALT_RUN to run taxprofiler pipeline in hasta.

### CODE REVIEW

**Review by jfy133 — CHANGES_REQUESTED** [review]
Just missing  corresponding`docs/` and adding the list to the `Pipeline-specific institutional documentation` section on the main README of the repository

File: conf/pipeline/taxprofiler/hasta.config [hunk 1]
```diff
@@ -0,0 +1,10 @@
+process {
```
Comment (jfy133) [inline conf/pipeline/taxprofiler/hasta.config:1]:
```suggestion
params {
  // Specific nf-core/configs params
  config_profile_contact = 'Sofia Stamouli (@sofstam)'
  config_profile_description = 'nf-core/taxprofiler HASTA profile provided by nf-core/configs'
}


process {
```

**Review by jfy133 — APPROVED** [review]
Just need update on main readme, and if your local tests pass you can merge :+1:

**Comment by sofstam** [comment]
Yes, the local tests pass.

**Comment by jfy133** [comment]
Merge away!

**Pull request merged**